docs/src/api/class-apirequestcontext.md

```diff
@@ -371,6 +371,16 @@ await request.GetAsync("https://example.com/api/getText", new() { Params = param
 * since: v1.16
 ### option: APIRequestContext.get.headers = %%-js-python-csharp-fetch-option-headers-%%
 * since: v1.16
+### option: APIRequestContext.get.data = %%-js-python-csharp-fetch-option-data-%%
+* since: v1.26
```

Comment:
I'd mark it `* since v1.27` since this is going to be the first version where typescript version supports it.